.github/PULL_REQUEST_TEMPLATE/01-bug-fix.md

```diff
@@ -1,3 +1,14 @@
+---
+name: 🐛 Bug fix
+about: >-
+  Submit a change to behavior the maintainers deemed undesired,
+  including regression tests
+title: Fix {{ X }} making it behave like {{ Y }}
+labels:
+- bug
```

Comment:
Totally. But this makes it a bit more responsive. I think it's a nice touch that could eventually lead to dropping this specific bit of classification from the bot. It's also more resilient since the submitters can't actually change these labels, meaning that the bot won't need to rely on heuristics of what text shows up in the issue body to know that the creator actually clicked that button for the correct category.